meta/store.go

```diff
@@ -735,12 +734,20 @@ func (s *Store) serveExecListener() {
 		// Accept next TCP connection.
 		var err error
 		conn, err := s.ExecListener.Accept()
-		if err != nil {
-			if strings.Contains(err.Error(), "connection closed") {
+		if opErr, ok := err.(*net.OpError); ok && opErr.Temporary() {
```

Comment:
If the error is temporary, then continue. Other error return. if error == nil , handle this Conn.
And there is no need to sleep for a second, deleted.